src/authenticationservices.cs

```diff
@@ -490,6 +506,10 @@ enum ASAuthorizationProviderAuthorizationOperation {
 		[DefaultEnumValue]
 		[Field (null)]
 		None,
+
+		[Mac (10, 16), iOS (14, 0)]
+		[Field ("ASAuthorizationProviderAuthorizationOperationConfigurationRemoved")]
+		Removed,
```

Comment:
Shouldn't this be `ConfigurationRemoved`?

That's what swift does: https://developer.apple.com/documentation/authenticationservices/asauthorizationproviderauthorizationoperation/3566366-configurationremoved